www/docs/en/dev/guide/platforms/android/webview.md

```diff
@@ -38,97 +38,56 @@ legacy `CordovaActivity` component that pre-dates the 1.9 release.
-            @Override
-            public void onCreate(Bundle savedInstanceState) {
-                super.onCreate(savedInstanceState);
-                setContentView(R.layout.main);
-                cwv = (CordovaWebView) findViewById(R.id.tutorialView);
-                Config.init(this);
-                cwv.loadUrl(Config.getStartUrl());
-            }
-
-1. If the application needs to use the camera, implement the
-   following:
-
-        @Override
-        public void setActivityResultCallback(CordovaPlugin plugin) {
-            this.activityResultCallback = plugin;
-        }
-        /**
-         * Launch an activity for which you would like a result when it finished. When this activity exits,
-         * your onActivityResult() method is called.
-         *
-         * @param command           The command object
-         * @param intent            The intent to start
-         * @param requestCode       The request code that is passed to callback to identify the activity
-         */
-        public void startActivityForResult(CordovaPlugin command, Intent intent, int requestCode) {
-            this.activityResultCallback = command;
-            this.activityResultKeepRunning = this.keepRunning;
-
-            // If multitasking turned on, then disable it for activities that return results
-            if (command != null) {
-                this.keepRunning = false;
-            }
-
-            // Start activity
-            super.startActivityForResult(intent, requestCode);
-        }
-
-        @Override
-        /**
-         * Called when an activity you launched exits, giving you the requestCode you started it with,
-         * the resultCode it returned, and any additional data from it.
-         *
-         * @param requestCode       The request code originally supplied to startActivityForResult(),
-         *                          allowing you to identify who this result came from.
-         * @param resultCode        The integer result code returned by the child activity through its setResult().
-         * @param data              An Intent, which can return result data to the caller (various data can be attached to Intent "extras").
-         */
-        protected void onActivityResult(int requestCode, int resultCode, Intent intent) {
-            super.onActivityResult(requestCode, resultCode, intent);
-            CordovaPlugin callback = this.activityResultCallback;
-            if (callback != null) {
-                callback.onActivityResult(requestCode, resultCode, intent);
-            }
-        }
-
-1. Finally, remember to add the thread pool, otherwise plugins
-   have no threads on which to run:
-
-        @Override
-        public ExecutorService getThreadPool() {
-            return threadPool;
-        }
+1. Follow the instructions here to build your first cordova app [CreateYourFirstApp](https://cordova.apache.org/docs/en/8.x/guide/cli/index.html)
```

Comment:
Why does one do that? Just to get the files to your harddisk from where you can copy them?

Reply:
My interpretation (and use case) was to embed a cordova app into another Android app that was developed with native code. As this is a getting started documentation, I assumed it's a good starting point to build a first app. Everyone having a working cordova app by hand will obviously skip the step. Does this make sense?

Reply:
Almost. What do you need the Cordova app for when actually embedding the Cordova Android webview in the native app?

Reply:
One of my customers has the requirement to develop two native apps within a larger project. He also needs to have geospatial capabilities (like Google maps) but without the restrictions of the popular SDK's provided by Mapbox, Google,... The organization that will be offering the apps has licenses for the "Hexagon Mapworks SDK". But this is more a ready-to-use app than a SDK so I needed to run it within the native target app.

Reply:
What I've seen and learned is that the cordova build applies some magic to the html sources when copying them into the platform(s). I didn't look into this further but without a build for android, no assets/www sources are available.
The same applies to all other files and folders as only the full cordova build (cordova create + cordova add platform + cordova build) creates the android project sources.

So the scenario(s) are two different ones:
1. User has a native app AND a working cordova app that needs to be integrated. (Described here)
2. User has nothing, wants to create a cordova app and then do some native work in the app

With two different starting points, it's quite difficult to described it in general :-)

Reply:
Yep - I have never build an app that uses Cordova in that way at all, that's why I am asking such dumb questions.